parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -461,6 +461,8 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
 {{- if EnableDataEncryptionAtRest }}
     sed -i "s|<etcdEncryptionSecret>|\"{{WrapAsParameter "etcdEncryptionKey"}}\"|g" /etc/kubernetes/encryption-config.yaml
 {{end}}
+    {{- /* Ensure that container traffic can't connect to internal Azure IP endpoint */}}
+    iptables -I FORWARD -d 168.63.129.16 -p tcp -j DROP
```

Comment:
you will need to specify port 80 as this IP is also used by Load Balancer probe.
ref PR: https://github.com/Azure/AgentBaker/pull/375